server/login-by-phone.js

```diff
@@ -1,15 +1,16 @@
 Meteor.methods({
   loginByPhone: function(code, phone){
-    user = Meteor.users.findOne({'services.phone.verificationTokens.code': code, 'profile.phone': phone});
+    user = Meteor.users.findOne({'services.phone.verificationTokens.code': code, 'profile.phone': phone}) ||
+      Meteor.users.findOne({'services.phone.verificationTokens.code': code, 'phone': phone});
 
```

Comment:
Actually the more I think about this we should probably make propName an argument that maps where the user has stored the phone number.  Something like

```
loginByPhone: function(code, phone, propName){
  user = Meteor.users.findOne({'services.phone.verificationTokens.code': code, propName: phone});
```

I hadn't thought about people have other mappings than the schema we were using for users.